docs/source/actions/queue.rst

```diff
@@ -358,6 +360,9 @@ These are the options of the ``queue`` action:
        * ``merge`` to merge the base branch into the pull request.
        * ``rebase`` to rebase the pull request against its base branch.
 
+       This option is ignored if ``method`` is set to ``fast-forward``,
+       ``rebase`` will be always used.
```

Comment:
```suggestion
       This option is ignored if ``method`` is set to ``fast-forward``:
       ``rebase`` will be always used.
```

Can we raise rather than ignore? We could change the default to `None` where it'd pick the right method by default depending on the merge method.

Reply:
done